build.sh

```diff
@@ -43,21 +43,36 @@ function buildImage {
 	# we are probably using zip with source code and we need to enter
 	# coreboot directory first
 		cd coreboot
-        else
+	else
 	# we are probably in the coreboot repository already
 		if [ ! -d 3rdparty/blobs/mainboard ]; then
 			git submodule update --init --checkout
 		fi
 	fi
 
-	if [ ! -d 3rdparty/blobs/mainboard/protectli ]; then
-		git clone git@github.com:Dasharo/protectli-blobs.git 3rdparty/blobs/mainboard/protectli
-	fi
+	if [ "$_platform" != "fw4c" ]; then
+
+		if [ ! -d 3rdparty/blobs/mainboard/protectli ]; then
+			git clone git@github.com:Dasharo/protectli-blobs.git 3rdparty/blobs/mainboard/protectli
```

Comment:
@miczyg1 Let's use HTTPS, it will be easier for users to build. Otherwise, they will get:

```
root@e73150f5df76:/# git clone git@github.com:Dasharo/protectli-blobs.git
Cloning into 'protectli-blobs'...
The authenticity of host 'github.com (140.82.121.4)' can't be established.
ED25519 key fingerprint is SHA256:+DiY3wvvV6TuJJhbpZisF/zLDA0zPMSvHdkr4UvCOqU.
This key is not known by any other names
Are you sure you want to continue connecting (yes/no/[fingerprint])? yes
Warning: Permanently added 'github.com' (ED25519) to the list of known hosts.
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

Reply:
@macpijan this path is not for FW4C. Cloning private repo with HTTPS will prompt for username and password. And possibly a 2FA?

Reply:
@macpijan ?

Reply:
Oh, right. For these older FW2/FW4 boards we would also only miss the ME to be extracted the same way as for FW4C? Or would there be more problems?

Reply:
@macpijan we had different FSP binaries for FW2 and FW4, but they were from public repo IIRC so not much of a problem here. But ME and IFD has to be extracted